### PR TITLE
Automate Npm Publish

### PR DESCRIPTION
**Pull request by puskuruk**
Automatic semantic release

### CODE REVIEW

**Review by mrdadah — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Build NPM package
+        run: npm run build
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
+      - name: Publish github pages
+        run: |
+          git remote set-url origin https://git:${GITHUB_TOKEN}@github.com/UniversalDataTool/universal-data-tool.git
```
Comment (seveibar) [inline .github/workflows/release.yml]: wrong repository

File: .releaserc.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    branch: "master",
+    plugins: [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      ["@semantic-release/npm", { npmPublish: false }],
```
Comment (seveibar) [inline .releaserc.js]: false publish isn't needed (can be removed)

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "test": "ava"
+    "test": "ava",
+    "publish": "npm publish"
```
Comment (seveibar) [inline package.json]: not important, but it's not good practice to have a "publish" script. If there are prepublish actions, they'll be executed automatically if a script called "prepublish" is created

**Review by seveibar — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (puskuruk) [reply .github/workflows/release.yml]: [Fixed](https://github.com/UniversalDataTool/udt-labelme-converter/pull/2/commits/6fa149a66ba6b3e12993026c28c82d90b4f1cf55)

File: package.json (hunk 3 above)
Comment (puskuruk) [reply package.json]: [Removed](https://github.com/UniversalDataTool/udt-labelme-converter/pull/2/commits/793c5b5ca975ddad4c46a546ea43ea236ccb78ab)

File: .releaserc.js (hunk 2 above)
Comment (puskuruk) [reply .releaserc.js]: [Removed](https://github.com/UniversalDataTool/udt-labelme-converter/pull/2/commits/793c5b5ca975ddad4c46a546ea43ea236ccb78ab)

**Review by seveibar — APPROVED** [review]

**Pull request merged**